### PR TITLE
Update styles to be like Figma

### LINKED ISSUE #165: Wrong autocomplete design
**Describe the bug**
Wrong autocomplete design, must be must be the same on figma

**To Reproduce**
1. Click on 'Explore'
2. Click on 'Create new action'
3. Use auto-complete (choose the verifiers input)

**Expected behavior**

Correct on Figma:
- Background;
- Same width of input;
- Full name first;
- Username (I liked this idea, you can add username below of full name);

Screen Shot 2020-02-17 at 15 33 45

https://www.figma.com/file/nEPG6Dkns8hDVCudVUTEzc/Dashboard?node-id=122%3A1217

Wrong on HTML:

Screen Shot 2020-02-17 at 15 31 07

**Cambiatus Env you are using**
app.cambiatus.io

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by marcjal**
## What issue does this PR close
Closes #165 

## Changes Proposed ( a list of new changes introduced by this PR)
This PR tends to refactor an actual view to be more like Figma

Captura de Tela 2021-02-02 às 19 21 23

## How to test ( a list of instructions on how to test this PR)
- Login to the platform
- Open transfer page
- type something like `kar` 
- See results

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -527,3 +527,7 @@ textarea.input {
 
 /* https://tailwindcss.com/docs/extracting-components#extracting-css-components-with-apply */
 @tailwind utilities;
+
+.elm-select-menu .elm-select-item:last-of-type .items-center > .border-dotted{
+  border-style: unset
+}
```
Comment (lucca65) [inline src/styles/main.css:533]: please add an empty line at the end of the file

**Review by lucca65 — CHANGES_REQUESTED** [review]

File: src/styles/main.css (hunk 1 above)
Comment (marcjal) [reply src/styles/main.css]: sure! updated

**Comment by marcjal** [comment]
@lucca65 an e-mail with a message that ci is failed was sent me, something about slack notification and a webhook

**Review by lucca65 — CHANGES_REQUESTED** [review]
@marcjal can you move the css classes to around line 104? 

We should add a section like this:

```css

/* Elm Select*/

.elm-select-menu .elm-select-item:last-of-type .items-center > .border-dotted {
   border-style: unset
 }

/* End of Elm Select*/
```

**Comment by marcjal** [comment]
> @marcjal can you move the css classes to around line 104?
> 
> We should add a section like this:
> 
> ```css
> /* Elm Select*/
> 
> .elm-select-menu .elm-select-item:last-of-type .items-center > .border-dotted {
>    border-style: unset
>  }
> 
> /* End of Elm Select*/
> ```

sure! let me update it

**Review by lucca65 — APPROVED** [review]

**Comment by lucca65** [comment]
@marcjal Thanks so much for the contribution! Its awesome to receive some help! Please let us know if you want to go  further deep with it, maybe we have some more challenging issues so you can work on your Elm skills

**Comment by marcjal** [comment]
> @marcjal Thanks so much for the contribution! Its awesome to receive some help! Please let us know if you want to go further deep with it, maybe we have some more challenging issues so you can work on your Elm skills

sure! let's do it xD i'd like to continue contributing

**Comment by marcjal** [comment]
for some reason, the build failed

**Comment by lucca65** [comment]
> for some reason, the build failed. this is normal? @lucca65

Don't worry ;)

**Pull request merged**